packages/hash/api/src/graphql/typeDefs/comment.typedef.ts

```diff
@@ -0,0 +1,112 @@
+import { gql } from "apollo-server-express";
+
+export const commentTypedef = gql`
+  type Comment implements Entity {
+    properties: CommentProperties!
+
+    contents: [TextToken!]!
```

Comment:
The idea was for it to have the same name as `Page.contents` for consistency, but in hindsight it might not be the best idea since  `Page.contents` isn't even linking to `tokens` anyway.
Do you think I should change the name to `tokens`?

Reply:
Tokens works, though you'd want consistency so you should make the argument name in the mutation match.

Reply:
Changed it to `tokens` in a1c877964a12ce6f89e923a8593aed07f7c0b0d4.